sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/view/ReflectiveViewRouter.scala

```diff
@@ -17,12 +17,15 @@
 package kalix.springsdk.impl.view
 
 import com.google.protobuf.any.{ Any => ScalaPbAny }
-import kalix.javasdk.Metadata
+import kalix.javasdk.{ JsonSupport, Metadata }
```

Comment:
two lines, one line? :)

Reply:
not sure what you mean :S would you elaborate?

Reply:
ah, you mean two separate lines?

I mean, I guess this should be automatic.. otherwise it's a pain.. I think this is fine if no checks are triggered. I checked that we have other instances in the code of using this format.